### PR TITLE
Feature/2304 ml attachment details additional fields

### PR DESCRIPTION
**Pull request by sagarsdeshmukh**
Changes -
- Added original language title, original language description and restrictions fields in media details panel
- Added toggle default and original title, desc option in media details panel

### CODE REVIEW

**Comment by comzeradd** [comment]
You probably need to rebase first. This seems to include commits already merged on the previous PR.

File: classes/api/class-media-image.php [hunk 1]
```diff
@@ -160,6 +160,42 @@ public function setPathTr2( $path_tr2 ) {
 		return $this;
 	}
 
+	/**
+	 * @return mixed
+	 */
+	public function getOriginalLanguageTitle() {
+		return $this->original_language_title;
```
Comment (angtheod) [inline classes/api/class-media-image.php]: Could you declare this field? Or there is a reason you want to use it dynamically?

File: classes/api/class-media-image.php [hunk 2]
```diff
@@ -160,6 +160,42 @@ public function setPathTr2( $path_tr2 ) {
 		return $this;
 	}
 
+	/**
+	 * @return mixed
+	 */
+	public function getOriginalLanguageTitle() {
```
Comment (angtheod) [inline classes/api/class-media-image.php]: @sagarsdeshmukh All methods in this class are named with camelCase, while we have been using under_score naming convention.

Comment (angtheod) [reply classes/api/class-media-image.php]: Also, I know it is not in this PR but [here](https://github.com/greenpeace/planet4-plugin-medialibrary/blob/cef8f71a176a87a15071181427323ae1afaaca7e/classes/api/class-media-image.php#L140) there are spaces missing after and before the parenthesis

File: classes/api/class-media-image-mapper.php [hunk 3]
```diff
@@ -22,7 +22,9 @@ public function get_from_array( array $params ) {
 			->setTitle( (string) $params['Title'] )
 			->setCaption( (string) $params['Caption'] )
 			->setCredit( isset( $params['Artist'] ) ? $params['Artist'] : '' )
-			->setPathTr1( (string) $params['Path_TR1']['URI'] );
+			->setPathTr1( (string) $params['Path_TR1']['URI'] )
+			->setOriginalLanguageTitle( isset( $params['original-language-title'] ) ? (string) $params['original-language-title'] : '' )
+			->setOriginalLanguageDesc( isset( $params['original-language-description'] ) ? (string) $params['original-language-description'] : '' );
```
Comment (angtheod) [inline classes/api/class-media-image-mapper.php]: could you use the null coalescing operator in the last 2 cases?

File: classes/api/class-media-image.php (hunk 1 above)
Comment (sagarsdeshmukh) [reply classes/api/class-media-image.php]: Yes I will declare those fields.

File: classes/api/class-media-image-mapper.php (hunk 3 above)
Comment (sagarsdeshmukh) [reply classes/api/class-media-image-mapper.php]: Yes, use of  null coalescing operator is good fit here.

**Comment by comzeradd** [comment]
This still includes the old commits, so let's make sure we squash them before we merge.

File: includes/ml_view.twig [hunk 4]
```diff
@@ -79,6 +82,26 @@
                                     id="ml-button-insert" disabled="disabled">{{ __( 'Insert into WP media & post', data.domain ) }}
                             </button>
                         </div>
+
+                        <div class="ml-additional-fields">
+                            <hr>
+                            <label class="ml-org-lang-label">{{ __( 'In original language:', data.domain ) }}</label>
```
Comment (kirdia) [inline includes/ml_view.twig]: @sagarsdeshmukh Please use hardcoded text-domain for translatable strings, as these won't be picked up by some tools (eg. LocoTranslate)

**Pull request merged**